schemas/neuromaps.schema.json

```diff
@@ -33,6 +37,35 @@
                                         },
                                         "sphere": {
                                             "$ref": "#/$defs/surfaceHemisphere"
+                                        },
+                                        "inflated": {
```

Comment:
Here, I would leave the `inflated` out since we can't assume every template will have an inflated surface. Instead, we should allow for `additionalProperties` at this level!